### ISSUE TITLE
Glad someone made this but, how do I start it?

### ISSUE THREAD

**Issue by MothgirlEnjoyer** [dentolos19/SmRecipeModifier#1]
IDK if I'm dumb or something but I can't for the life of me find a way to actually start this thing as its own programme.
I've gone through every file I could open and there's no .exe or .bat files that could start it. Tried opening a few of the files that would open in Visual Studio Code but the most I've used VSC for is making simple html websites for school so I'd have no idea how to start it from there if that's even possible.

Maybe you could add some instructions to the readme?

Once again IDK maybe I'm just stupid

Any response would be greatly appreciated!

**Comment by dentolos19** [comment]
It's okay. It is also my bad to not include some instructions on how to get started using this program.

Here's the way to download and use it...

1. Download the latest package from the [releases](https://github.com/dentolos19/RecipeMechanic/releases) page.
2. Extract and run `RecipeMechanic.exe`. If it doesn't run, try installing the latest [.NET Desktop Runtime](https://dotnet.microsoft.com/download/dotnet).
3. Click `File` and then `Open` and navigate to the game's executable. Most likely your executable is located at `C:\Program Files (x86)\Steam\steamapps\common\Scrap Mechanic\Release`.
4. Select a recipe file to edit, once done, click `File` and then `Save` to save your changes.
5. Done!

> **Note**: It is recommended to backup the game first before starting, so in the program, click `Game` and then `Steam Backup Wizard` and proceed with the instructions given by the wizard.

I tried to explain it as simple as possible. If there's any further questions, ask right away.

And also, it's been awhile since I've updated this program. You can create a new issue regarding any problems with the program and I will try to resolve it as soon as possible.

**Comment by MothgirlEnjoyer** [comment]
Thank you so much! I don't use GitHub so I don't even know how to navigate to the releases page lol.

This makes so much more sense than what I had downloaded earlier.

Thank you for the speedy response too, very cool!
